### PR TITLE
Generalize steps

### PR DESCRIPTION
**Pull request by geoffreydhuyvetters**
### Changed

- Created a more generic PlaceStep
- Create a EventTypeAndThemStep
- Set defaults for movie entry form

### CODE REVIEW

File: src/pages/manage/movies/MovieForm.tsx [hunk 1]
```diff
@@ -309,16 +332,12 @@ const MovieForm = () => {
 
   const createNewEvent = async ({
     production,
-    cinema,
-    theme: themeId,
+    place,
+    eventTypeAndTheme: { eventType, theme },
```
Comment (brampauwelyn) [inline src/pages/manage/movies/MovieForm.tsx:336]: imho I think `eventCategoryAndTheme` would be a better name.
eventType confuses me with offerType...

File: src/pages/manage/movies/MovieForm.tsx [hunk 2]
```diff
@@ -154,9 +161,25 @@ const convertSubEventsToTimeTable = (subEvents: SubEvent[] = []) => {
...
     resolver: yupResolver(schema),
+    defaultValues: {
+      timeTable: {
+        data: {},
+        dateStart: formatDate(nextWeekWednesday),
+        dateEnd: formatDate(nextWeekWednesday),
+      },
+      eventTypeAndTheme: {
+        eventType: {
+          id: '0.50.6.0.0',
```
Comment (brampauwelyn) [inline src/pages/manage/movies/MovieForm.tsx:178]: Use constant `OfferCategories.Film` from `OfferCategories.ts`

**Review by brampauwelyn — CHANGES_REQUESTED** [review]

File: src/pages/manage/movies/MovieForm.tsx (hunk 1 above)
Comment (geoffreydhuyvetters) [reply src/pages/manage/movies/MovieForm.tsx:336]: EventType is the actual naming though

https://publiq.stoplight.io/docs/uitdatabank/ZG9jOjUwNjExMQ-taxonomy-terms

Comment (geoffreydhuyvetters) [reply src/pages/manage/movies/MovieForm.tsx:336]: Screenshot 2022-03-10 at 15 00 29

Comment (brampauwelyn) [reply src/pages/manage/movies/MovieForm.tsx:336]: 👍

**Review by brampauwelyn — APPROVED** [review]

File: src/ui/TimeTable.tsx [hunk 3]
```diff
@@ -303,18 +296,6 @@ const TimeTable = ({ id, className, onChange, value, ...props }: Props) => {
...
-      const nextWeekWednesday = nextWednesday(new Date());
-      onChange({
-        data: value?.data ?? {},
-        dateStart: formatDate(nextWeekWednesday),
-        dateEnd: formatDate(nextWeekWednesday),
-      });
-    }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
-
```
Comment (simon-debruijn) [inline src/ui/TimeTable.tsx:317]: Why was this removed, isn't this needed?

Comment (geoffreydhuyvetters) [reply src/ui/TimeTable.tsx:317]: I've moved this to the defaultValues of the form.
